perception/compare_map_segmentation/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ ament_target_dependencies(compare_map_segmentation
   sensor_msgs
   tier4_autoware_utils
   autoware_map_msgs
+  autoware_adapi_v1_msgs
```

Comment:
@isamu-takagi Is it ok that adapi msgs is used in component?

Reply:
@yukkysaito did you mean replace autoware_adapi_v1_msgs through [component_interface_specs/localization.hpp](https://github.com/autowarefoundation/autoware.universe/blob/main/common/component_interface_specs/include/component_interface_specs/localization.hpp)

Reply:
@yukkysaito I changed here https://github.com/autowarefoundation/autoware.universe/pull/3277/commits/44948e57674a9ef60cc90027f30ea534ac181d95

Reply:
@yukkysaito I fixed localization interface utilize https://github.com/autowarefoundation/autoware.universe/pull/3277/commits/da31686778654b0bea61cdcc44279e90dffb1b42